client/scss/layouts/_form-side.scss

```diff
@@ -8,13 +8,12 @@
       w-right-0
```

Comment:
Should this be in `components` instead of `layouts`? 🤔

Reply:
Indeed!

Reply:
Has been flagged on a future issue to fix up https://github.com/wagtail/wagtail/issues/8867